hortum_mobile/lib/data/announ_data_backend.dart

```diff
@@ -6,18 +6,23 @@ import 'package:hortum_mobile/globals.dart';
 class AnnounDataApi {
   List<dynamic> announcements = [];
 
-  Future getAnnoun() async {
+  Future getAnnoun(String filter) async {
     //Trocar o IPLOCAL pelo ip de sua máquina
     String userAccessToken = await actualUser.readSecureData('token_access');
-    var url = 'http://$ip:8000/announcement/list';
+    String url;
+    if (filter.isEmpty)
+      url = 'http://$ip:8000/announcement/list';
+    else
+      url = 'http://$ip:8000/announcement/list/${filter}';
+
     var header = {
       "Content-Type": "application/json",
       "Authorization": "Bearer " + userAccessToken,
     };
 
     var response = await http.get(url, headers: header);
+    this.announcements = [];
```

Comment:
Só uma dúvida mesmo, o que esse `this.announcements = [];` está fazendo?

Reply:
Bem observado, escrevi essa linha pra fazer um teste e acabei esquecendo de apagá-la. Já corrigi aqui.